File: 백준/Silver/18187. 평면 분할/평면 분할.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int []num= new int[101];
        num[1]=2;
        num[2]=4;
        int count = 3;
        for(int i=3;i<=n;i++){
            num[i]=num[i-1]+count;
            if(i%3!=0){
                count+=1;
            }
        }
        System.out.print(num[n]);
    }
}

File: 백준/Silver/18187. 평면 분할/평면 분할.js
const fs = require('fs');
const input = fs.readFileSync("/dev/stdin").toString().trim();
var n = Number(input);
var num = [];
num[1]=2;
num[2]=4;
var count = 3;
for(let i =3;i<=n;i++){
  num[i]=num[i-1]+count;
  if(i%3!=0){
    count++;
  }
}
console.log(num[n]);